x-pack/plugins/observability_solution/slo/common/typings.ts

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Any reason to name this file typings instead of types? I don't see us using this naming elsewhere.

Reply:
Fixed